### PR TITLE
3.0: Drop deprecated features and use inclusive language

### PR DESCRIPTION
**Pull request by enrico-usai**
* Remove every reference to Torque and SGE schedulers
* Remove jobwatcher, sqswatcher and nodewatcher code.
* Remove nodewatcher, sqswatcher, jobwatcher references.
* Remove ASG utilities.
* Use inclusive language: from master to head node.

### CODE REVIEW

File: src/common/utils.py [hunk 1]
```diff
@@ -306,7 +260,7 @@ def _get_instance_info(region, proxy_config, instance_type, additional_instance_
 
 def get_instance_properties(region, proxy_config, instance_type, additional_instance_types_data=None):
```
Comment (demartinofra) [inline src/common/utils.py:261]: we could also remove this logic since not used in slurm plugin if I recall correctly

Comment (enrico-usai) [reply src/common/utils.py]: Done, I removed also other unused utility functions.

File: tox.ini [hunk 2]
```diff
@@ -15,8 +15,7 @@ deps =
     -r tests/requirements.txt
     cov: codecov
 commands =
-    bash ./tests/test.sh
-    nocov: python -m unittest discover -s tests/jobwatcher -p "*tests.py"
+    nocov: python -m unittest discover -p "*tests.py"
```
Comment (demartinofra) [inline tox.ini:18]: we don't need this cause the only unit test which was not in pytest has been removed

**Review by demartinofra — CHANGES_REQUESTED** [review]
LGTM! one remaining minor thing and then good to go

File: tox.ini (hunk 2 above)
Comment (enrico-usai) [reply tox.ini]: Removed, thanks

**Review by demartinofra — APPROVED** [review]

**Pull request merged**